subprojects/gradle/impact/src/main/kotlin/com/avito/impact/plugin/ImpactMetricsSender.kt

```diff
@@ -0,0 +1,28 @@
+package com.avito.impact.plugin
+
+import com.avito.android.sentry.EnvironmentInfo
+import com.avito.android.stats.GaugeMetric
+import com.avito.android.stats.StatsDSender
+import com.avito.android.stats.graphiteSeries
+import com.avito.impact.ConfigurationType
+
+class ImpactMetricsSender(
```

Comment:
I think we should track that only for CI builds.
When we don't have a teamcityBuildId?

Reply:
added preconditions:

```
 require(environmentInfo.environment is Environment.CI)
        { "ImpactMetricsSender should run only in CI environment" }

        requireNotNull(environmentInfo.teamcityBuildId())
        { "ImpactMetricsSender should run only if teamcityBuildInfo available" }
```

refactoring of environment info is a big task for this PR